editor/src/messages/portfolio/document/overlays/grid_overlays.rs

```diff
@@ -135,9 +171,16 @@ pub fn overlay_options(grid: &GridSnapping) -> Vec<LayoutGroup> {
 				RadioEntryData::new("isometric")
 					.label("Isometric")
 					.on_update(update_val(grid, |grid, _| grid.grid_type = GridType::ISOMETRIC)),
+				RadioEntryData::new("dot")
+					.label("Dot")
+					.on_update(update_val(grid, |grid, _| grid.grid_type = GridType::DOT)),
```

Comment:
The dot grid works well, but I think that it should not be displayed as a seperate grid type (as it is merely an alternative rendering method for the rectangle grid). Probably a check box if the rectangle grid enabled would work better.